ams/server/src/test/java/com/netease/arctic/server/table/executor/TestOrphanFileCleanIceberg.java

```diff
@@ -42,4 +71,67 @@ public TestOrphanFileCleanIceberg(CatalogTestHelper catalogTestHelper, TableTest
     super(catalogTestHelper, tableTestHelper);
   }
 
+  @Test
+  public void cleanIndependentFiles() throws IOException {
+    List<Record> records = Lists.newArrayListWithCapacity(3);
+    records.add(tableTestHelper().generateTestRecord(1, "test1", 0, "2022-01-01T00:00:00"));
+    List<DataFile> dataFiles1 = tableTestHelper().writeBaseStore(getArcticTable().asUnkeyedTable(), 1L,
+        records, false);
+    UnkeyedTable testTable = getArcticTable().asUnkeyedTable();
+    AppendFiles appendFiles = testTable.newAppend();
+    dataFiles1.forEach(appendFiles::appendFile);
+    appendFiles.commit();
+    assertIndependentFiles(testTable, 0);
+    SortedPosDeleteWriter<Record> posDeleteWriter = AdaptHiveGenericTaskWriterBuilder.builderFor(testTable)
+        .buildBasePosDeleteWriter(0, 0, dataFiles1.get(0).partition());
+    posDeleteWriter.delete(dataFiles1.get(0).path(), 0);
+    List<DeleteFile> posDelete = posDeleteWriter.complete();
+    testTable.newRowDelta().addDeletes(posDelete.get(0)).commit();
+    records.clear();
+    records.add(tableTestHelper().generateTestRecord(3, "test3", 0, "2022-01-02T00:00:00"));
+    List<DataFile> dataFiles2 = tableTestHelper().writeBaseStore(getArcticTable().asUnkeyedTable(), 1L,
+        records, false);
+    testTable.newRewrite().rewriteFiles(Collections.singleton(dataFiles1.get(0)),
+            Collections.singleton(dataFiles2.get(0)))
+      .validateFromSnapshot(testTable.currentSnapshot().snapshotId()).commit();
+    assertIndependentFiles(testTable, 1);
+    OrphanFilesCleaningExecutor.cleanIndependentFiles(testTable);
+    assertIndependentFiles(testTable, 0);
+  }
+
+  private void assertIndependentFiles(UnkeyedTable unkeyedTable, int count) {
+    TableScan tableScan = unkeyedTable.newScan();
+    Set<String> files = new HashSet<>();
+    for (FileScanTask task : tableScan.planFiles()) {
+      files.add(task.file().path().toString());
```

Comment:
task file don't need add in set

Reply:
Thx，the task file is added to the sets to determine the use of independent files.

Reply:
AFAIK, Only delete file will be independent files?

Reply:
Yes, but this is a validation method for unit testing, and performing more rigorous checks does not hurt.

Reply:
ok